test/k8s/simple_test.go

```diff
@@ -70,7 +70,28 @@ func Simple() {
 	})
 
 	AfterEach(func() {
-		time.Sleep(stabilizationSleep)
+		endpoint := cluster.GetPortForward("prometheus-server").ApiServerEndpoint
+		promClient, err := NewPromClient(fmt.Sprintf("http://%s", endpoint))
+		Expect(err).ToNot(HaveOccurred())
+
+		stopCh := make(chan struct{})
+		metricCh := make(chan int)
+		errCh := make(chan error)
+
+		go WatchXdsDeliveryCount(promClient, stopCh, metricCh, errCh)
+		defer close(stopCh)
+
+	Loop:
+		for {
+			select {
+			case <-metricCh:
```

Comment:
should this value be used? otherwise might as well be `struct{}`? but doesn't matter